test/e2e/network/networking.go

```diff
@@ -508,18 +508,6 @@ var _ = SIGDescribe("Networking", func() {
 
 	})
 
-	// Once basic tests checking for the sctp module not to be loaded are implemented, this
-	// needs to be marked as [Disruptive]
-	ginkgo.It("should function for pod-pod: sctp [Feature:SCTPConnectivity][Disruptive]", func() {
-		config := e2enetwork.NewNetworkingTestConfig(f, e2enetwork.EnableSCTP)
-		ginkgo.By(fmt.Sprintf("dialing(sctp) %v --> %v:%v (config.clusterIP)", config.TestContainerPod.Name, config.ClusterIP, e2enetwork.ClusterSCTPPort))
-		message := "hello"
-		err := config.DialEchoFromTestContainer("sctp", config.TestContainerPod.Status.PodIP, e2enetwork.EndpointSCTPPort, config.MaxTries, 0, message)
```

Comment:
I don't think we need to delete this test, just update it to use `config.EndpointPods` as the targetIP?

Reply:
> This is just paralleling the tcp and udp tests. I think the right fix is to fix the test case, not to remove it

^ what Dan said :)

I understand we want dashboard green, but this seems like a quick/easy fix now?

Reply:
It took me a while to found them, the pod to pod should go here, with the other pod to pod 
https://github.com/kubernetes/kubernetes/pull/96460#issuecomment-725491063